workers/data_refinery_workers/downloaders/utils.py

```diff
@@ -27,31 +29,37 @@ def start_job(job: DownloaderJob):
     job.save()
 
 
-def end_job(job: DownloaderJob, batch: Batch, success):
-    """Record in the database that this job has completed,
-    create a processor job, and queue a processor task."""
-    job.success = success
-    job.end_time = timezone.now()
-    job.save()
-
+def end_job(job: DownloaderJob, batches: Batch, success):
+    """Record in the database that this job has completed.
+    Create a processor job and queue a processor task for each batch
+    if the job was successful."""
```

Comment:
I do have flycheck and autopep8 both running haha. Since I've written this I've read up more on doc string conventions though so I've already started doing it more correctly in my new code. I'll definitely fix this up though.